float32[] Right_Joint_Readings
float32[] Left_Joint_Readings